docs/functionality/ipc_tinker.md

```diff
@@ -0,0 +1,45 @@
+# IPC slot value tinker
+
+Each ipc slot offers a dbus api interface.
+One for filters, get value and value tinkering.
+
+## General information
+The slot requests a name from dbus `<slot_name>._slot_`
+storing its dbus connection, 
+The implemented functionality inject their interfaces
+into the slots established dbus connection.
+
+### Interface names
+the following interface names are used for the functionality.
+
+- filters `<slot_name>/filters`
+- tinker  `<slot_name>/tinker`
+- value `<slot_name>/value`
+
+## Specification of the tinker interface
+property called `forced` of the type bool,
+this property indicates weather the slots value is forced or not.
+
+a method called `force` accepting a value of the type of the slot,
+this method forces the value of the slot.
+
+a method called `release` accepting no arguments.
```

Comment:
What does `release` do?

Reply:
release is unforce or releasing the force of the value.

Reply:
Specified in doc.